paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt

```diff
@@ -114,33 +114,45 @@ internal object ImageUtils {
     }
     assertEquals(TYPE_INT_ARGB.toLong(), goldenImage.type.toLong())
 
-    val imageWidth = Math.min(goldenImage.width, image.width)
-    val imageHeight = Math.min(goldenImage.height, image.height)
+    val goldenImageWidth = goldenImage.width
+    val goldenImageHeight = goldenImage.height
+
+    val imageWidth = Math.max(goldenImageWidth, image.width)
+    val imageHeight = Math.max(goldenImageHeight, image.height)
 
     // Blur the images to account for the scenarios where there are pixel
     // differences
     // in where a sharp edge occurs
     // goldenImage = blur(goldenImage, 6);
     // image = blur(image, 6);
-
-    val width = 3 * imageWidth
+    val width = goldenImageWidth + imageWidth + image.width
```

Comment:
Did you also try what happens if the image gets smaller on either/both dimensions?

Reply:
Yeah, in that scenario it basically looks like the images in the description except the goldenImage is larger and the image on the right is smaller